modules/renter/contractor/host_integration_test.go

```diff
@@ -104,9 +105,9 @@ func newTestingContractor(testdir string, cs modules.ConsensusSet, tp modules.Tr
 	return New(cs, w, tp, hdb, filepath.Join(testdir, "contractor"))
 }
 
-// newTestingTrio creates a Host, Contractor, and TestMiner that can be used
-// for testing host/renter interactions.
-func newTestingTrio(name string) (modules.Host, *Contractor, modules.TestMiner, error) {
+// newTestingTrio a Contractor, TestMiner, and a slice of hosts that can be
+// used for testing host/renter interactions.
+func newTestingTrio(name string, nhosts int) ([]modules.Host, *Contractor, modules.TestMiner, error) {
```

Comment:
I agree that this change is ugly, I'll revert it and create an extra host in `TestIntegrationSetAllowance`.